test/recur_iterator_test.js

```diff
@@ -879,4 +879,47 @@ suite('recur_iterator', function() {
       '2019-01-07T08:00:00'
     ]
   });
+
+  //yearly, byYearDay
+  testRRULE('FREQ=YEARLY;BYYEARDAY=1,2,4,6,11,12,20,42,48,49,-306,-303,' +
+            '-293,-292,-266,-259,-258,-239,-228,-209,-168,-164,-134,-133,' +
+            '-113,-105,-87,-56,-44,-26,-21,-14', {
+    dtStart: '2015-01-01',
+    dtEnd: '2015-01-02',
+    max: 32,

```

Comment:
`dtEnd` is not handled (nor needed) and `max` is defaults to the length of dates if not specified. Can you remove the lines on `dtEnd` and `max`?